Makefile

```diff
@@ -75,9 +76,10 @@ manifests: crd
 
 # Generate CRD yaml from our api types:
 .PHONY: crd
-crd: ensure-controller-gen
+crd: ensure-controller-gen ensure-yq
 	rm -rf ./config/crds
 	'$(CONTROLLER_GEN)' crd paths=./pkg/apis/hive/v1 output:dir=./config/crds
+	yq -y -i . ./config/crds/*.yaml
```

Comment:
how does this work? I don't see a mention of `-y` in the yq docs. Is it quiet mode? And does this just make yq do a reformat?

Reply:
The `-y` flag makes the output yaml instead of json.

Reply:
yq reads in the file then outputs it as yaml and replaces the file with the output.

Reply:
ok cool. I have to use `r` for read in the version I have. Shrug. lgtm!

Reply:
The issue is there are multiple versions of yq out there that work slightly differently.

Reply:
Yes, you have a completely different yq.

Reply:
Ah, I am using the wrong one in the makefile.